app/controllers/subscriptions_management_controller.rb

```diff
@@ -118,4 +118,13 @@ def is_single_page_subscription?(subscription)
     subscription.dig("subscriber_list", "content_id").present?
   end
   helper_method :is_single_page_subscription?
+
+  def get_heading_content(subscription)
+    if is_single_page_subscription?(subscription)
+      "<a href='#{subscription['subscriber_list']['url']}' class='govuk-link'>#{subscription['subscriber_list']['title']}</a>"
+    else
+      subscription["subscriber_list"]["title"]
+    end
+  end
+  helper_method :get_heading_content
```

Comment:
Yeah this looks good.
If we decide to go with moving this to a helper [over here](https://github.com/alphagov/email-alert-frontend/pull/1197#discussion_r751131532). Perhaps we could merge that one then rebase and let this one join it too?

In which case... I should test this, but I think because our method gets invoked from a view we should be able to make that line 124 use a `link_to` method (https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to).

The syntax on that can be a pain, but I think your class goes in the options hash.

If we did it that way... I have s suspicion we won't need to use santize... I think...

Reply:
Agreed. I've moved the other one out into the helper so once that's in I'll rebase this branch and move this method out as well.

I'll have a look at `link_to` when I do that

Reply:
Ok! I've done the refactor and `link_to` wasn't too bad either. 

I also took the opportunity to move the 'you subscribed at' into the locale file